bigframes/ml/llm.py

```diff
@@ -184,12 +185,25 @@ def predict(
         # Params reference: https://cloud.google.com/vertex-ai/docs/generative-ai/learn/models
         if temperature < 0.0 or temperature > 1.0:
             raise ValueError(f"temperature must be [0.0, 1.0], but is {temperature}.")
-        if max_output_tokens not in range(1, 1025):
+
+        if (
+            self.model_name == _TEXT_GENERATOR_BISON_ENDPOINT
+            and max_output_tokens not in range(1, 1025)
+        ):
+            raise ValueError(
+                f"max_output_token must be [1, 1024] for TextBison model, but is {max_output_tokens}."
+            )
+        elif (
```

Comment:
Done, this was in case any new model may be added, and it would be easier to see what current cases are, but just use else is also reasonable I guess.

Reply:
Done.